scripts/lint-style.lean

```diff
@@ -155,7 +155,15 @@ def undocumentedScripts (opts : Lean.Options) : IO Nat := do
 
 /-- Implementation of the `lint-style` command line program. -/
 def lintStyleCli (args : Cli.Parsed) : IO UInt32 := do
-  let opts ← getLakefileLeanOptions
+  -- Use the environment declared in Mathlib.Tactic.Linter.TextBased to determine the linter sets.
+  Lean.initSearchPath (← Lean.findSysroot)
+  let sets ← unsafe Lean.withImportModules #[{module := `Mathlib.Tactic.Linter.TextBased}] {}
+    fun env => pure <| linterSetsExt.getState env
+
+  let opts : LinterOptions := {
+    toOptions := ← getLakefileLeanOptions,
+    linterSets := sets,
+  }
```

Comment:
I can't review this part of the PR or the change in `MathlibTest/ModuleCasing.lean` but other than that LGTM. 

bors d+